src/components/Title.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Title = styled.div`
+  display: flex;
+  justify-content: center;
+  font-size: ${({ fontSize }) => fontSize}px;
+`;
+export default ({ title, fontSize }) => {
+  return <Title fontSize={fontSize}>{title}</Title>;
+};
```

Comment:
[참고](https://stackoverflow.com/questions/34676984/cannot-export-const-arrow-function) 이렇다고 하더라구요!
이렇게 바꾸는게 더 좋을 것 같습니당

```suggestion
const votePageTitle= ({ title, fontSize }) => {
  return <Title fontSize={fontSize}>{title}</Title>;
};
export default votePageTitle;
```

Reply:
오오 저도 알아갑니다 !